packages/plugins/documentation/server/middlewares/restrict-access.js

```diff
@@ -9,7 +9,7 @@ module.exports = async (ctx, next) => {
     return next();
   }
 
-  if (!ctx.session.documentation || !ctx.session.documentation.logged) {
+  if (!ctx.session?.documentation || !ctx.session?.documentation.logged) {
```

Comment:
With optional chaining you can safely combine them:

```suggestion
  if (!ctx.session?.documentation?.logged) {
```